docs/content/reference/make-commands.md

```diff
@@ -17,26 +17,21 @@ Generates the code for the downstream `google` and `google-beta` providers.
 Examples:
 
 ```bash
-make provider VERSION=ga OUTPUT_PATH="$GOPATH/src/github.com/hashicorp/terraform-provider-google"
-make provider VERSION=beta OUTPUT_PATH="$GOPATH/src/github.com/hashicorp/terraform-provider-google-beta"
+go run . --version ga --output $GOPATH/src/github.com/hashicorp/terraform-provider-google
```

Comment:
I think this command will actually stay the same. It'll just call the `go run` command under the hood. 

Basically we just want to modify any references to `bundle exec`